scripts/connection-tester.js

```diff
@@ -4,11 +4,10 @@ var net = require('net');
 
 var socket = new net.Socket(),
     host = process.argv[2],
-    port = process.argv[3];
+    port = process.argv[3],
+    connectTimeout = process.argv[4];
 
-var SOCKET_TIMEOUT = 500;   //Setting 500ms as max acceptable timeout
-
-socket.setTimeout(SOCKET_TIMEOUT);
+socket.setTimeout(connectTimeout);
```

Comment:
it will be taken care from here.

i thought of having one place to hold a default value.

https://github.com/chunkiat82/node-connection-tester/blob/master/index.js#L81

what do you think?